Mnemnk App doesn't start after installing mnemnk-lifelogging

### Describe the bug
After importing lifelogging.json in mnemnk-lifelogging, Mnemnk App stops with the following error.

```
[2025-05-26][06:52:30][mnemnk_app_lib][ERROR] Failed to initialize agent: stream did not contain valid UTF-8
```

### To Reproduce
Steps to reproduce the behavior:
1. Install mnemnk-lifelogging
2. Import lifelogging.json
3. Start the flow. (At this moment, agents in the flow works)
4. Save the flow.
5. Restart Mnemnk App/

### Expected behavior
A clear and concise description of what you expected to happen.

### Screenshots
If applicable, add screenshots to help explain your problem.

### Context
- Version v0.18.8
- OS: macOS Sequoia 15.5

--- a/src-tauri/src/mnemnk/agent/env.rs
+++ b/src-tauri/src/mnemnk/agent/env.rs
@@ -178,11 +178,8 @@ impl AgentEnv {
             if new_name.starts_with('/') || new_name.ends_with('/') || new_name.contains("//") {
                 return false;
             }
-            // Disallow segments that are "." or ".."
-            if new_name
-                .split('/')
-                .any(|segment| segment == "." || segment == "..")
-            {
+            // Disallow segments that starts with "."
+            if new_name.split('/').any(|segment| segment.starts_with('.')) {
                 return false;
             }
         }

--- a/src-tauri/src/mnemnk/agent/flow.rs
+++ b/src-tauri/src/mnemnk/agent/flow.rs
@@ -172,6 +172,10 @@ fn read_agent_flows_recursive<P: AsRef<Path>>(
                     .trim_end_matches(".json")
                     .replace('\\', "/"); // Ensure consistent path separators
 
+                if flow_name.starts_with('.') {
+                    continue;
+                }
+
                 // Process JSON files
                 let flow = read_agent_flow(flow_name.clone(), path.clone())?;
 

